packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -81,7 +81,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
             rowStyleProvider: (row: TimelineChart.TimeGraphRowModel) => ({
                 backgroundColor: 0x979797,// 0xaaaaff,
                 backgroundOpacity: row.selected ? 0.1 : 0,
-                lineColor: 0xdddddd, // hasStates ? 0xdddddd : 0xaa4444, // row.data && row.data.hasStates
+                lineColor: 0x35373B , // hasStates ? 0xdddddd : 0xaa4444, // row.data && row.data.hasStates
```

Comment:
These are only an improvement in dark theme, in light theme it increases the contrast, so the grid line colors should probably be theme dependent...

Reply:
I agree with @PatrickTasse that the color should depend on dark or light theme. Personally, I find the colors of the gridlines to dark and not visible, especially the vertical gridlines.

Reply:
I changed the colors with what Stefan said.